[#1286] ci: helm chart to deploy a dev-latest operator

lavocatt opened the pull request:
In order to let people try easily the latest change of the operator we now provide a dev.latest helm chart.

brusdev reviewed (commented):
Can you move the changes for adding the imagePullPolicy field to another commit/issue?

lavocatt committed:
[#1290] helm charts: introducing pullPolicy

In order to let users customize the behavior of their deployment, it
make sense to introduce a pullPolicy in our helm charts. With using a
default value of null in the base chart we won't impose any particular
behavior, but instead opening a nice customization path for everyone who
needs it.

lavocatt committed:
[#1286] ci: helm chart to deploy a dev-latest operator

In order to let people try easily the latest change of the operator we
now provide a dev.latest helm chart.
